test/unit/pipeline/test_predict_sklearn.py

```diff
@@ -83,20 +86,35 @@ def test_pre_process_input_data(
         for mock_func in [nonzero_flatliner_mock, replace_invalid_data_mock]:
             self.assertEqual(mock_func.call_count, 1)
 
-    # TODO include the test below to see if the pipeline uses the fallback forecast correctly
-    # This test requires a functioning 'happy_flow_pipeline' test which is not yet developed
-    @unittest.skip(
-        "No functioning happy_flow_pipeline test available. Add this one later"
+    @patch(
+        "openstf.pipeline.predict_sklearn.MODEL_LOCATION", Path("./test/trained_models")
     )
     @patch("openstf.validation.validation.is_data_sufficient")
     def test_incomplete_input(self, is_data_sufficient_mock):
         """Test if a fallback forecast is used when input is incomplete"""
```

Comment:
Same as my previous comment. It can be hulpful to easily determine which test is for which unit by using consistent naming. So in this case `test_predict_pipeline_incomplete_inputdata` could be more suitable.

Reply:
I'll change it, good suggestion!